django-s3-file-field-client/src/client.ts

```diff
@@ -120,11 +134,12 @@ export default class S3FFClient {
    * Uploads a file using multipart upload.
    *
    * @param file the file to upload
+   * @param fieldId the django model field's id
    * @param options the upload options
    */
-  public async uploadFile(file: File, fieldId: string): Promise<UploadResult> {
+  public async uploadFile(file: File, fieldId: string, options: UploadOptions = {}): Promise<UploadResult> {
```

Comment:
I'd rather pass this to the `S3FFClient` constructor. It's not something that would change for each file uploaded (and if it is, the user can simply create a new `S3FFClient` each time)